src/components/NewConnectionModal/index.tsx

```diff
@@ -32,14 +37,20 @@ const NewConnectionModal: React.FC<ModalProps> = ({ visible, ...rest }) => {
   const handleTestConnection = useCallback(() => {
     if (!formRef.current) { return }
 
-    const { host, port, password } = formRef.current.getData() as ConnectionFormData
+    const { host, port, password, sshHost, sshPort, sshUser, sshPassword } = formRef.current.getData() as ConnectionFormData
```

Comment:
What do you think about put ssh options into sub object. I mean
```
interface SshOptions {
  host: string;
  port: number;
  user: string;
  password: string
}

interface ConnectionFormData {
  host: string;
  port: number;
  password: string;
  sshOptions?: SshOptions
}
```

And then put inputs for ssh options under accordion? ([what accordion is?](https://material-ui.com/components/accordion/#accordion))

Reply:
I think it's a good idea, but the purpose of this PR is to connection by SSH Tunnel. The ideal was to create a roadmap with the next steps, and provide the template (zeplin.io), or similar, what do you think @diego3g ?

Reply:
I'll create, but take some time, it's a side project so i don't have enough time to create all the documentation, layout and tasks. I'll try to take the monday night to do these stuff. But thanks a lot for all of the contribution.